src/SFProvider.cc

```diff
@@ -71,14 +77,14 @@ TH1F* SFProvider::LoadHistogram(TFile& file, std::string_view name)
 float SFProvider::FindBinValue(const TH1F& hist, float x, int unc_scale)
 {
     int bin = hist.FindFixBin(x);
-    bin = std::clamp(bin, 1, hist.GetNbinsX());
+    //bin = std::clamp(bin, 1, hist.GetNbinsX());
```

Comment:
Why did you comment out this line? It is used to map the scale factor for pTs outside the histogram range to the last available bin that is properly filled.

Reply:
Actually, I forgot to uncomment out. I was checking what exactly the line is doing.